support/jsdoc/theme/static/styles/jsdoc-default.css

```diff
@@ -127,12 +143,32 @@ section, h1 {
 
 .nav.navbar-right .navbar-form {
     padding: 0;
-    margin: 0;
+    margin: 6px 0px;
+}
+
+.navbar-collapse.collapse {
+  display: block!important;
+}
+
+.navbar-nav>li, .navbar-nav {
+  float: left !important;
+}
+
+.navbar-nav.navbar-right:last-child {
+  margin-right: -15px !important;
+}
+
+.navbar-right {
+  float: right!important;
```

Comment:
I guess that the layout can be done, and then re-written without bootstrap. I can help with that later, but would be good to set the 'target' layout now, so that later, it is only migrated...